travel_hr/i18n/fr.po

```diff
@@ -19,9 +19,9 @@ msgstr ""
 "X-Poedit-SourceCharset: UTF-8\n"
 
 #. module: travel_hr
-#: field:travel.passenger,department_id:0 field:travel.travel,department_id:0
-msgid "Department"
-msgstr "Unité administrative"
+#: field:travel.travel,department_rule:0
+msgid "Department Rule"
+msgstr "Filtre UI"
```

Comment:
This is the same translation as line 44 this one might be the wrong one. Shouldn't it be UA instead of UI?

As at line 39